engine/api/workflow/dao_node_run.go

```diff
@@ -92,6 +93,13 @@ func LoadNodeRun(db gorp.SqlExecutor, projectkey, workflowname string, number, i
 		}
 		r.Coverage = cov
 	}
+	if loadOpts.WithVulnerabilities {
```

Comment:
I could be nice to have unit tests to cover this option

Reply:
done